config/initializers/business_time.rb

```diff
@@ -0,0 +1,6 @@
+BusinessTime::Config.load("#{Rails.root}/config/business_time.yml")
+
+# or you can configure it manually:  look at me!  I'm Tim Ferriss!
```

Comment:
we probably don't need these comments

Reply:
the comments are so people in the future now they can either amend the config using the yaml file or doing it manually, we have other config files that are nothing but comments so I thought it made sense to keep them?